Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,92 @@
+# Preparing development environment and first rkt build
+This is an example configuration for a fresh installation on Ubuntu 16.04 GNOME.
+This doc is also a short quick start guide.
+For a detailed reference see [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Get rkt repo and install dependencies
```

Comment:
Each heading needs a blank line before and after it.
This should be a level two heading (`##`).